packages/universal-cookie/package.json

```diff
@@ -3,13 +3,22 @@
   "version": "7.0.0",
   "description": "Universal cookies for JavaScript",
   "main": "cjs/index.js",
+  "types": "cjs/index.d.ts",
+  "type": "module",
```

Comment:
The type module will break CJS builds. This is why we use `.mjs` files for ESM. This line will need to be removed.

Reply:
Yes, I doesn't have this problem on m'y others project because I name cjs files with .cjs . 

I will fix it when updating thé others packages